### PR TITLE
Reliable transaction submission diagram

### PR DESCRIPTION
**Pull request by mDuo13**
Adds a diagram for reliable transaction submission and some additional instructions for how to use the account Sequence to sanity check when transactions are not successfully processed (thanks @intelliot for the idea there).

You can preview the updated document here: https://developers-staging.ripple.com/reliable-transaction-submission.html#best-practices (Ripple VPN required)

Notes: 

- The `.uxf` file is the source file for the diagram and can be ignored.
- The `.svg` diagram is generated from the `.uxf` source and used as the display version. This vector file should render smoothly on a wide variety of devices and sizes.

### CODE REVIEW

File: content/tutorials/use-simple-xrp-payments/reliable-transaction-submission.md [hunk 1]
```diff
@@ -122,9 +126,13 @@ For each persisted transaction without validated result:
         If (server has continuous ledger history from the ledger when the
               transaction was submitted up to and including the ledger
               identified by LastLedgerSequence)
-            The transaction failed (2)
-            If appropriate for the application, submit with
-                new LastLedgerSequence and Fee
+
+            # Sanity check
+            If (sender account sequence > transaction sequence)
+                A different transaction with this sequence has a final outcome.
```
Comment (nbougalis) [inline content/tutorials/use-simple-xrp-payments/reliable-transaction-submission.md:132]: Should this be _potentially_ different transaction? Perhaps the transaction executed and you didn't notice. I guess it depends on how you intend "_If (result appears in validated ledger)_" to be interpreted. Does it mean "in _a_ validated ledger" or "in the _newest_ validated ledger"?

**Review by nbougalis — COMMENTED** [review]
Looks good. Left a comment that might help clarify the text a bit.

**Comment by mDuo13** [comment]
!rebuild

File: content/tutorials/use-simple-xrp-payments/reliable-transaction-submission.md (hunk 1 above)
Comment (mDuo13) [reply content/tutorials/use-simple-xrp-payments/reliable-transaction-submission.md:132]: Ah, yes, "If (result appears in validated ledger)" should be "If (result appears in any validated ledger)"

Will update that. :+1:

File: content/tutorials/use-simple-xrp-payments/reliable-transaction-submission.md [hunk 2]
```diff
@@ -137,15 +145,30 @@ The difference between the two transaction failure cases (labeled (1) and (2) in
...
 
-- In failure case (2), the transaction was not included in a validated ledger, so it did nothing at all, not even destroy the transaction cost. This could be the result of the transaction cost being too low for the current load on the XRP Ledger, the `LastLedgerSequence` being too soon, or it could be due to other conditions such as an unstable network connection. In contrast to failure case (1), it is more likely that a new transaction is likely to succeed if you change only the `LastLedgerSequence` and possibly the `Fee` and submit again.
+- In failure case (2), the transaction was not included in a validated ledger, so it did nothing at all, not even destroy the transaction cost. This could be the result of the transaction cost being too low for the current load on the XRP Ledger, the `LastLedgerSequence` being too soon, or it could be due to other conditions such as an unstable network connection.
+
+    - In contrast to failure case (1), it is more likely that a new transaction is likely to succeed if you change only the `LastLedgerSequence` and possibly the `Fee` and submit again. However, you should not do this
```
Comment (jhaaaa) [inline content/tutorials/use-simple-xrp-payments/reliable-transaction-submission.md]: nit edit: add a period to the end of "you should not do this"

File: content/tutorials/use-simple-xrp-payments/reliable-transaction-submission.md [hunk 3]
```diff
@@ -137,15 +145,30 @@ The difference between the two transaction failure cases (labeled (1) and (2) in
...
+
+    - If you have two or more transaction-sending systems in an active/passive failover configuration, it's possible that the passive system mistakenly believes the active system has failed, and has become active while the original active system is still also sending transactions. Check the connectivity between the systems and ensure that at most one of them is active. Check your account's transaction history (for example, with the [account_tx method][]) and record the final outcome of all transactions that were included in validated ledgers. Any different transactions with the same `Sequence` numbers have failed permanently; save those final outcomes as well. When you have finished reconciling the differences from all the systems and have resolved the issues that made the systems activate simultaneously, resume normal activities.
+
+        **Tip:** The [`AccountTxnID` field](transaction-common-fields.html#accounttxnid) can help prevent redundant transactions from succeeding in this situation.
```
Comment (jhaaaa) [inline content/tutorials/use-simple-xrp-payments/reliable-transaction-submission.md:162]: 💎

**Review by jhaaaa — APPROVED** [review]
Thank you for this detailed and informative content, Rome.

**Pull request merged**